R/copy-model-from.R

```diff
@@ -147,18 +150,19 @@ copy_model_from.numeric <- function(
 
 #' Copy model from an existing NONMEM model
 #'
+#' Private implementation function called by `copy_model_from()` dispatches.
```

Comment:
General comment: no need to specify that a function is "private." `@keywords internal` will take care of that, and if in the future you decide to export the function, now you have to make two changes rather than one.